src/main/target/SITL/target.c

```diff
@@ -60,15 +60,21 @@ const timerHardware_t timerHardware[1]; // unused
 uint32_t SystemCoreClock;
 
 static fdm_packet fdmPkt;
+static rc_packet rcPkt;
 static servo_packet pwmPkt;
+static servo_packet_raw pwmRawPkt;
+
+static bool rc_received = false;
+static bool fdm_received = false;
 
 static struct timespec start_time;
 static double simRate = 1.0;
-static pthread_t tcpWorker, udpWorker;
+static pthread_t tcpWorker, udpWorker, udpWorkerRC;
 static bool workerRunning = true;
-static udpLink_t stateLink, pwmLink;
+static udpLink_t stateLink, pwmLink, pwmRawLink, rcLink;
 static pthread_mutex_t updateLock;
 static pthread_mutex_t mainLoopLock;
+static char simulator_ip[32] = "127.0.0.1";
```

Comment:
(the idea was that you can use pointer to string - either use constant string from this line, or change pointer to command line argument (it is valid until program ends). But it is minor problem, possibly just add `strncpy(,,sizeof(simulator_ip))` to targetParseArgs() )